server/plugin/utils.go

```diff
@@ -325,7 +334,11 @@ func getToDoDisplayText(baseURL, title, url, notifType string) string {
 	if len(title) > 80 {
 		title = strings.TrimSpace(title[:80]) + "..."
 	}
-	titlePart := fmt.Sprintf("[%s](%s)", title, url)
+
+	titlePart = fmt.Sprintf(": %s", title)
+	if url != "" {
+		titlePart = fmt.Sprintf("[%s](%s)", title, url)
+	}
```

Comment:
2/5 that we want consistency here. All variants of `titlePart` should either have a `:` or not.